azconfig/client/src/test/java/com/azure/azconfig/AzConfigTest.java

```diff
@@ -41,6 +35,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Properties;
+import java.util.concurrent.atomic.AtomicInteger;
```

Comment:
I am not seeing the usageof this class, do we need this import?

Reply:
I pushed up a commit that cleans up any unused imports. :D